### ISSUE TITLE
New LEDC auto channel function needs ability to manual adjust channel.

### ISSUE THREAD

**Issue by spyder0069** [espressif/arduino-esp32#8967]
### Board

ESP32

### Device Description

ESP32

### Hardware Configuration

transducer and backlight

### Version

latest development Release Candidate (RC-X)

### IDE Name

Arduino IDE

### Operating System

Windows 10

### Flash frequency

40Mhz

### PSRAM enabled

no

### Upload speed

115200

### Description

The new function got rid of the setup call that allowed you to manually set your channel.  The problem is my project has a audio transducer and a lcd with backlight.  When using ledc for tones and controlling the backlight brightness the two interfere.  This is because the esp groups two sets of eight channels and the frequency is shared amongst the set.  Since it now auto adds the channels its grouping both of my devices and when the frequency changes for tone control the backlight is then off and you can't fix this without a blinking in the backlight while they are grouped.  Previously I could set channel 1 for backlight and 9 for the speaker and everything was happy since the frequencies were seperated.

### Sketch

```cpp
ledcAttach(buzzer,2000,8);
ledcAttach(TFT_BACKLIGHT,2000,8);
```

### Debug Message

```plain
No debug messages
```

### I have checked existing issues, online documentation and the Troubleshooting Guide

- [X] I confirm I have checked existing issues, online documentation and Troubleshooting guide.

**Comment by spyder0069** [comment]
I did find my work around.  I found some others talking about how each two consecutive channels share a timer.  I thought the first 8 shared one and the second 8 shared another but I was wrong.  4 timers spread out.  It was mentioned here "github.com/esphome/issues/issues/3114"  

My workaround was to take a unused pin and use it as a unused ledc channel.  
`
myledcresponse=ledcAttach(buzzer,2000,8); 
myledcresponse=ledcAttach(39,2000,8); //  create a false channel with unused input.  
myledcresponse=ledcAttach(TFT_BACKLIGHT,2000,8);
`
It would be nice if the option to manually select the channel would be returned to this function in the future.
